src/main/scala/vexriscv/demo/VexRiscvAhbLite3.scala

```diff
@@ -122,12 +122,8 @@ object VexRiscvAhbLite3{
               mscratchGen    = false,
               mcauseAccess   = CsrAccess.READ_ONLY,
               mbadaddrAccess = CsrAccess.READ_ONLY,
-              mcycleAccess   = CsrAccess.NONE,
```

Comment:
Would it be possible to still keep those option ?
I know that would be redundant to the CounterPlugin implementation, but that would preserve backward compatibility without any disruption. (not break users setups)

Reply:
(this is a general comment, so would also apply for the other demo)

Reply:
The options are still available in the CSRPlugin. I removed them from here and other places, since CsrAccess.NONE was and is the default value for these.

Reply:
Would be possible to revert all the removals of "mcycleAccess = CsrAccess.NONE" and similar things ? 

They are there to make it clear to the user that it isn't included, and that if they want they can just turn it on there.